### PR TITLE
chore: import translations from openedx-translations

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

We are manually importing translation files from openedx-translations while openedx-atlas is in place.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by Ian2012** [comment]
@OmarIthawi how can I compile the `.mo` files?

**Review by OmarIthawi — APPROVED** [review]
Thanks @Ian2012!

**Comment by OmarIthawi** [comment]
> @OmarIthawi how can I compile the `.mo` files?

@Ian2012 run `i18n_tools generate --verbose` or `python manage.py compilemessages`

**Comment by Ian2012** [comment]
@OmarIthawi ready, but translations will not work till the `translate` template tag is used correctly. This PR depends on: https://github.com/openedx/FeedbackXBlock/pull/41

I will wait till that PR is merged and the translation file updated to do this contribution

**Comment by OmarIthawi** [comment]
@Ian2012 how about `python manage.py compilemessages`, does that work?

`i18n_tools generate` calls `compilemessages` under the hood, so we probably don't need `i18n_tool generate`

you could also compile via `msgcat` which is the low-level tool of `compilemessage`.

Regardless of the method, this pull request needs `.mo` files, otherwise it won't work.

**Comment by Ian2012** [comment]
@OmarIthawi The `.mo` are in the PR but the translations are not working correctly as the templates are not using the `translate` template tag

**Review by OmarIthawi — APPROVED** [review]
> @OmarIthawi The `.mo` are in the PR but the translations are not working correctly as the templates are not using the `translate` template tag

You're right! I'm sorry I missed that! It looks great now. Please reset the tests because codecov has failed.

**Comment by Ian2012** [comment]
@OmarIthawi tests are successful now

File: feedback/locale [hunk 1]
```diff
@@ -0,0 +1 @@
+feedback/locale
```
Comment (OmarIthawi) [inline feedback/locale:1]: I don't think this link is needed. Please remove it.

We need the following link though:

`cd feedback && ln -sd conf/locale translations`

Similar to this one: https://github.com/openedx/xblock-drag-and-drop-v2/blob/master/drag_and_drop_v2/translations

**Review by OmarIthawi — CHANGES_REQUESTED** [review]
One required change and it should be ready to go. Thanks @Ian2012 

Please let me know when it's ready for review again.

File: feedback/locale (hunk 1 above)
Comment (OmarIthawi) [reply feedback/locale]: The symlink path needs to be updated. please run the following commands to create the right symlink:

```
cd FeedbackXBlock  # repo root
cd feedback  # package root
rm conf/locale/locale
ln -sd conf/locale translations
```

This is the only way translations can work.

Comment (OmarIthawi) [reply feedback/locale]: I don't see the link in the commits. please try again.

Try squashing all the changes and push force. 

I see the add6ce5 commit adds the link, but it doesn't appear of the Files tab of the PR https://github.com/openedx/FeedbackXBlock/pull/45/files

Something is wrong in the git commit

Comment (Ian2012) [reply feedback/locale]: I've squashed the commits and still not working. The translations simlink exists in the repository: https://github.com/openedx/FeedbackXBlock/blob/master/feedback/translations

**Comment by mphilbrick211** [comment]
Hi @feanil - just checking to see if this can be merged?

**Comment by feanil** [comment]
Tagging @OmarIthawi  and asking him to re-review since he was looking at this and is more familiar with the translations work.

**Comment by OmarIthawi** [comment]
@Ian2012 I'm sorry to let you know that we'll have to close this pull request.

As of Redwood, we no longer commit `.po` files into repositories. This pull request is still not working so, please consider hosting the translations in your own fork.

Please consider checking [OEP-58](https://docs.openedx.org/en/latest/developers/concepts/oep58.html) for more information about the new translation workflow.

cc: @mphilbrick211 @feanil

**Pull request closed without merging**